src/commands/function-list.json

```diff
@@ -20,7 +20,7 @@
             "type": "array",
             "items": {
                 "type": "object",
-                "additionalProperties": false,
+                "additionalProperties": true,
```

Comment:
Why? from the code it seems like we can have only the below properties.

Reply:
we don't want to commit to the number of fields in somewhat volatile places like FUNCTION LIST, MEMORY STATS etc.